modules/@apostrophecms/seo-fields-doc-type/index.js

```diff
@@ -1,47 +1,100 @@
+const _ = require('lodash');
+
 module.exports = {
   improve: '@apostrophecms/doc-type',
   fields(self, options) {
-    if (options.seoFields !== false) {
-      return {
-        add: {
-          seoTitle: {
-            label: 'aposSeo:title',
-            type: 'string',
-            htmlHelp: 'aposSeo:titleHtmlHelp'
-          },
-          seoDescription: {
-            label: 'aposSeo:description',
-            type: 'string',
-            htmlHelp: 'aposSeo:descriptionHtmlHelp'
-          },
-          seoRobots: {
-            label: 'aposSeo:robots',
-            htmlHelp: 'aposSeo:robotsHtmlHelp',
-            type: 'checkboxes',
-            choices: [
-              {
-                label: 'aposSeo:robotsNoFollow',
-                value: 'nofollow'
-              },
-              {
-                label: 'aposSeo:robotsNoIndex',
-                value: 'noindex'
-              }
-            ]
-          }
+    if (!options.seoFields) {
+      return;
+    }
```

Comment:
undefined would be an issue

can be related to https://github.com/apostrophecms/testbed/actions/runs/4495674703/jobs/7909607161